src/js/components/Drop/stories/Multiple.js

```diff
@@ -1,71 +1,63 @@
 import React, { useRef, useState } from 'react';
 
-import {
-  Box,
-  Button,
-  Drop,
-  DropButton,
-  Grommet,
-  Layer,
-  TextInput,
-} from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Button, Drop, DropButton, Layer, TextInput } from 'grommet';
 
 const MultipleDrop = () => {
   const [showDrop, setShowDrop] = useState(false);
   const [showLayer, setShowLayer] = useState(false);
   const targetRef = useRef();
 
   return (
-    <Grommet theme={grommet} full>
```

Comment:
@jcfilben just pointing out another case of `full`, as you mentioned it on another PR.

Reply:
Looking at this story I think we could either have the `full` prop or add padding to the box container just so the buttons are not at the very top of the screen.